docs/UserGuide.md

```diff
@@ -288,9 +348,9 @@ Format: `find KEYWORDS`
 
 <div markdown="span" class="alert alert-primary">
 **:bulb: Tip:** <br>
-* Use student name/telegram handle/consultation/mastery check date/remark to search up a particular student.
-* Multiple keywords can be entered and each keyword is separated by a space.
-* When searching for exam grade, use the format `find [exam_name:EXAM_SCORE]`.
+Use student name/telegram handle/consultation/mastery check date/remark to search up a particular student.
+Note: Multiple keywords can be entered and each keyword is separated by a space.
+Note: when searching for exam grade use format `find [exam_name:EXAM_SCORE]`
```

Comment:
I think it's better to follow the old format, as it been added properly as bullet points